pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -451,8 +451,8 @@ func TestReconcile_InvalidPipelineRuns(t *testing.T) {
 		))),
 		tb.PipelineRun("pipeline-invalid-final-graph", tb.PipelineRunNamespace("foo"), tb.PipelineRunSpec("", tb.PipelineRunPipelineSpec(
 			tb.PipelineTask("dag-task-1", "taskName"),
-			tb.FinalPipelineTask("final-task-1", "taskName"),
-			tb.FinalPipelineTask("final-task-1", "taskName")))),
```

Comment:
This error does not trigger an invalid dag when going through validation, it fails first on `validatePipelineTaskName`